docs/fxa-engineering/fxa-webchannel-protocol.md

```diff
@@ -0,0 +1,324 @@
+---
+id: fxa-webchannel-protocol
+title: WebChannel Protocol
+sidebar_label: WebChannel Protocol
+---
+
+Current as of `December 2nd, 2019`
+
+# Communication with Firefox via WebChannels
+
+Starting with Firefox 40, Firefox Accounts communicates with Firefox desktop and Fennec using [WebChannels](https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/WebChannel.jsm). WebChannels provide secure two-way communication between the browser and web content. Messages sent from content to the browser utilize `WebChannelMessageToChrome` custom events. Messages sent from the browser to content utilize `WebChannelMessageToContent` custom events.
```

Comment:
I wonder if Firefox 40 is sufficiently long ago that we don't need to talk about the Time Before WebChannels in this doc any more.

Reply:
Probably.. We officially only support down to 60 right now anyways.